src/main/kotlin/blackjack/factory/PlayerFactory.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.factory
+
+import blackjack.model.participant.Player
+import blackjack.validator.Validator
+
+interface BasePlayerFactory {
+    fun createPlayers(): List<Player>
+}
```

Comment:
```suggestion
fun interface BasePlayerFactory {
    fun createPlayers(): List<Player>
}
```
We can make this a functional interface if we have only one method in it.
It would be great to check out what functional interface is :)

Reply:
As I understand it so far, a `fun interface` allows us to pass a lambda.
I’m not exactly sure how I would adapt that in the current version. 
But if I ever need a new kind of test, I’ll be able to experiment with a lambda instead of creating a new class. right?

For now, I’ve just converted the `interface` to a `fun interface`.
Please have a look, and let me know if you’d recommend writing a test to try it out!

Reply:
Yes, you can use it in lambda 👍
That's it! You don't need to do anything.